nautobot/ipam/tests/test_models.py

```diff
@@ -295,6 +296,43 @@ def setUp(self):
         self.child1 = Prefix.objects.create(prefix="101.102.0.0/26", status=self.status, namespace=self.namespace)
         self.child2 = Prefix.objects.create(prefix="101.102.0.64/26", status=self.status, namespace=self.namespace)
 
+    def test_location_queries(self):
+        PrefixFactory.create_batch(10)
```

Comment:
Why's this needed? Don't we already have enough prefixes?

In general I've seen issues with using `Factory.create_batch()` inline in our unit tests when they are run in combination with `--cache-test-fixtures`, so I'd prefer to avoid this if possible.

Reply:
Some lines above all prefix was deleted

Reply:
```
 test_location_queries (nautobot.ipam.tests.test_models.TestPrefix.test_location_queries) failed:

    ValidationError({'__all__': ['VLAN with this Vlan group and ID already
    exists.']})
```

Reply:
Might need to manually create relevant prefixes for this specific test instead.